app/src/main/res/layout/dialog_forgot_password.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I assume you're using `CardView` so you can set color and corner radius. But you could use a shape xml drawable and set it as background for `LinearLayout` with the same result and remove `CardView` parent for less nesting